### DISCUSSION TITLE
was it necessary to use the name python-wasm for that part of your project?

### DISCUSSION THREAD

**Discussion by metaperl** [sagemathinc/cowasm#32] [Ideas]
Hello, while discussing [the differences between "python-wasm" and Pyodide](https://github.com/sagemathinc/zython/blob/main/docs/differences-from-pyodide.md) you stated NOTE: There is [another-python-wasm](https://github.com/ethanhs/python-wasm) 

My questions are:

1. Did that project pre-date yours?
2. have you considered using a different name, such as `zython-wasm`, or `wasm-python`  or `python-wasm-zig`?
3. How do you feel about 2 different projects having the same name?

**Comment by williamstein** [comment]
> 1. Did that project pre-date yours?

Yes, though I didn't find out about it until a while after using the name python-wasm.

> 2. have you considered using a different name, such as zython-wasm, or wasm-python or python-wasm-zig?

Yes, in fact I'm trying to rename it just "zython".  E.g., this very repo was called python-wasm before and I renamed it to "zython".  

My plan is only to use the name python-wasm inside my own monorepo, and maybe also @zython/python-wasm will reflect that subpackage to npm.

> 3. How do you feel about 2 different projects having the same name?

I don't think the name python-wasm is useful for a random build of python to wasm; there are dozens of such things.  It's far too generic for this particular project.   That's why I decided to not use it.  

I assume you are the one wanted to use it.  Are you specifically asking:

1. For me to transform the npmjs python-wasm name to you?
2. For me to transform the domain name python-wasm.org to you?

If so, definitely I'm open to either or both if you want to use python-wasm.

**Comment by williamstein** [comment]
It looks like we'll be using "CoWasm" for the name of this project overall soon.  It's a name that doesn't seem to be used for anything anywhere at all related to software (e.g., searching I found Zython is also used by something on Pypi already), and CoWasm as a name is somewhat connected with a motivating project (CoCalc).  

https://www.npmjs.com/package/cowasm
